core/store/src/metrics/mod.rs

```diff
@@ -583,6 +583,24 @@ pub static STORAGE_MISSING_CONTRACTS_COUNT: LazyLock<IntCounterVec> = LazyLock::
     .unwrap()
 });
 
+pub static ROCKS_ITERATOR_COUNT: LazyLock<IntCounter> = LazyLock::new(|| {
+    try_create_int_counter("near_rocksdb_iterators_total", "Number of rocksdb iterators created")
+        .unwrap()
+});
+
+pub static ROCKS_ITERATOR_LIVE_TIME: LazyLock<Counter> = LazyLock::new(|| {
```

Comment:
I suggest combining those 2 into single histogram metric, this way we will have both `_sum` and `_count` as well as some insights into latency distribution with `_bucket`

Reply:
You can just add 3 buckets let's say at 100ms, 1s, 30s, this way we will be able to see outliers and it won't add too much costs. I'd still argue that having just these 2 metrics won't be enough to detect rare outliers: if 99.999% is within 100us and then you have one very long-lived iterator it won't be visible on avg.